Core1Platform/AdministratingCore1/05_EngineRoom.md

```diff
@@ -0,0 +1,239 @@
+
+   ![Engine room](../../Assets/Screenshots/Core1Platform/AdministratingCore1/Engineroom.png "[Engine room]")
+
+2. Click the [PLATTFORM-KONSOLE] (Platform console) button.   
+   The *Platform Console* window is opened.
+
+   ![Platform console](../../Assets/Screenshots/Core1Platform/AdministratingCore1/EngineroomPlatformConsole.png "[Platform console]")
+
+3. Check the messages.
+
+
+
+## Check vCores
+
+Your company has purchased a certain number of vCores assigned the main account. vCores (CPU core voltage) define the computing power of an instance for the execution of parallel processes as well as database queries. Idle times are not considered, for example calls waiting for a response from external service providers.
+If an instance has not enough vCore capacity, the processor then starts to run the processes and database queries one after the other instead of running them in parallel. This means that the processing speed is slowed down significantly if the number of vCores assigned is unsufficiently.  
+
+> [INFO] The vCores define the power for running parallel processes. They do not define the power for running a single process. Assigning more vCores will not speed up single processes!    
+
+If a sandbox is created, it is assigned a number of vCores from the main account. This reduces the number of vCores available for the main account. Note, the transfer of vCores from the main account to a sandbox may take some time (about 5 minutes).  
+Actindo has an eye on the capacity utilization of your vCores. Isolated overruns will have no other effect than slowing down the speed of your processes. But if your vCore capacity is permanently over the limit, Actindo will contact you.   
+
+The engine room allows you to monitor the required vCores of the current instance. With the vCore dashlet, you can monitor the needs of the last two hours.  
+The vCores are analyzed as average over several seconds. The dashlet displays the totals per every 30 seconds in a curve diagram. 
+ > [INFO] Since Actindo displays an average value every 30 seconds, it may happen that the limit is exceeded for a brief time but is not displayed in the dashlet. This leads to a temporary slowdown of the processing speed.
+
+<!---hallo Julian, gibt es auch eine Standard-Statistik über einen längeren Zeitraum?-->
+
+#### Prerequisites
+
+- You have administrator rights for the current instance.
+
+#### Procedure
+
+*Actindo Core1 Platform > Any workspace*
+
+![Core1 Platform](../../Assets/Screenshots/Core1Platform/Core1.png "[Core1 Platform]")
+
+1. Click the ![Engine room](../../Assets/Icons/EngineRoom.png "[Engine roome]") (Engine room) button center top.  
+   The engine room is opened. The current workspace is overlapped. 
+
+   ![Engine room](../../Assets/Screenshots/Core1Platform/AdministratingCore1/Engineroom.png "[Engine room]")
+
+2. Focus your attention to the dashlet at the first position from the left.  
```

Comment:
Vielleicht kann man Schritt 2 in Schritt 3 integrieren, das dieser nicht so aussagekräftig ist, oder zwischen 1 und 3 verteilen... If finde die Aussage "Focus your attention..."  im Schritt irgendwie unüblich.